mkdocs.yml

```
@@ -5,6 +5,7 @@
 site_name: "lazy-importing"
 site_description: "Super-easy lazy importing in Python."
 site_url: "https://lazy-importing.readthedocs.io/en/latest/"
+repo_name: bswck/lazy_importing
```

Opinionated way of YAML-ing. I'm doing this suggestion just to apply it in this ticket.
```suggestion
repo_name: "bswck/lazy_importing"
```